README.md

```diff
@@ -334,11 +322,187 @@ class ApplicationController < ActionController::Base
 end
 ```
 
-## 3. Testing
+## 3. Quick Setup
```

Comment:
I don't like the idea of having both a "slow setup" and a "quick setup". I'd prefer to have a single "quick-ish setup", and then follow-up topics with more details. Topics could be organized by MVC layer, or by feature (passwords, crypto providers, magic columns, etc.). I'm not sure which topical organization scheme is better, but it should be consistent.